Ivy broken

When I clone a fresh copy of Joshua and type "ant", I get the following ivy-related errors. @lukeorland, can you look into this ASAP?

    $ git clone git@github.com:joshua-decoder/joshua joshua2
    $ cd joshua2
    $ export JOSHUA=$(pwd)
    $ ant
    Buildfile: /Users/post/code/joshua2/build.xml
      [taskdef] Could not load definitions from resource org/doxygen/tools/antlib.xml. It could not be found.
      [taskdef] Could not load definitions from resource testngtasks. It could not be found.
    
    setup:
      [taskdef] Could not load definitions from resource org/doxygen/tools/antlib.xml. It could not be found.
      [taskdef] Could not load definitions from resource testngtasks. It could not be found.
    
    check-joshua-home:
         [echo] JOSHUA = /Users/post/code/joshua2 basedir = /Users/post/code/joshua2
    
    download-ivy:
          [get] Destination already exists (skipping): /Users/post/code/joshua2/lib/ivy.jar
    
    init-ivy:
    
    resolve:
    [ivy:configure] :: Ivy 2.2.0 - 20100923230623 :: http://ant.apache.org/ivy/ ::
    [ivy:configure] :: loading settings :: file = /Users/post/code/joshua2/lib/ivysettings.xml
    [ivy:resolve] :: resolving dependencies :: joshua#joshua;working@fragment.local
    [ivy:resolve] 	confs: [default]
    [ivy:resolve] :: resolution report :: resolve 982ms :: artifacts dl 1ms
    	---------------------------------------------------------------------
    	|                  |            modules            ||   artifacts   |
    	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
    	---------------------------------------------------------------------
    	|      default     |   15  |   0   |   0   |   0   ||   0   |   0   |
    	---------------------------------------------------------------------
    [ivy:resolve] 
    [ivy:resolve] :: problems summary ::
    [ivy:resolve] :::: WARNINGS
    [ivy:resolve] 		::::::::::::::::::::::::::::::::::::::::::::::
    [ivy:resolve] 		::          UNRESOLVED DEPENDENCIES         ::
    [ivy:resolve] 		::::::::::::::::::::::::::::::::::::::::::::::
    [ivy:resolve] 		:: net.sourceforge.ant-doxygen#ant-doxygen;1.6.1: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='net.sourceforge.ant-doxygen' found='joshua'; bad module name: expected='ant-doxygen' found='joshua'; 
    [ivy:resolve] 		:: asm#asm;3.1: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='asm' found='joshua'; bad module name: expected='asm' found='joshua'; 
    [ivy:resolve] 		:: com.amazonaws#aws-java-sdk;1.1.3: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='com.amazonaws' found='joshua'; bad module name: expected='aws-java-sdk' found='joshua'; 
    [ivy:resolve] 		:: org.apache.commons#commons-cli;1.2: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='org.apache.commons' found='joshua'; bad module name: expected='commons-cli' found='joshua'; 
    [ivy:resolve] 		:: commons-logging#commons-logging;1.1.1: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='commons-logging' found='joshua'; bad module name: expected='commons-logging' found='joshua'; 
    [ivy:resolve] 		:: jaxen#jaxen;1.1.1: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='jaxen' found='joshua'; bad module name: expected='jaxen' found='joshua'; 
    [ivy:resolve] 		:: net.sf.jung#jung-algorithms;2.0: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='net.sf.jung' found='joshua'; bad module name: expected='jung-algorithms' found='joshua'; 
    [ivy:resolve] 		:: net.sf.jung#jung-api;2.0: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='net.sf.jung' found='joshua'; bad module name: expected='jung-api' found='joshua'; 
    [ivy:resolve] 		:: net.sf.jung#jung-graph-impl;2.0: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='net.sf.jung' found='joshua'; bad module name: expected='jung-graph-impl' found='joshua'; 
    [ivy:resolve] 		:: net.sf.jung#jung-visualization;2.0: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='net.sf.jung' found='joshua'; bad module name: expected='jung-visualization' found='joshua'; 
    [ivy:resolve] 		:: org.apache.hadoop#hadoop-core;0.20.203.0: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='org.apache.hadoop' found='joshua'; bad module name: expected='hadoop-core' found='joshua'; 
    [ivy:resolve] 		:: org.testng#testng;6.7: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='org.testng' found='joshua'; bad module name: expected='testng' found='joshua'; 
    [ivy:resolve] 		:: org.mockito#mockito-all;1.9.5: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='org.mockito' found='joshua'; bad module name: expected='mockito-all' found='joshua'; 
    [ivy:resolve] 		:: pmd#pmd;4.2.5: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='pmd' found='joshua'; bad module name: expected='pmd' found='joshua'; 
    [ivy:resolve] 		:: junit#junit;4.10: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='junit' found='joshua'; bad module name: expected='junit' found='joshua'; 
    [ivy:resolve] 		::::::::::::::::::::::::::::::::::::::::::::::
    [ivy:resolve] :::: ERRORS
    [ivy:resolve] 		filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='net.sourceforge.ant-doxygen' found='joshua'
    [ivy:resolve] 		filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='ant-doxygen found='joshua'
    [ivy:resolve] 		filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='asm' found='joshua'
    [ivy:resolve] 		filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='asm found='joshua'
    [ivy:resolve] 		filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='com.amazonaws' found='joshua'
    [ivy:resolve] 		filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='aws-java-sdk found='joshua'
    [ivy:resolve] 		filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='org.apache.commons' found='joshua'
    [ivy:resolve] 		filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='commons-cli found='joshua'
    [ivy:resolve] 		filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='commons-logging' found='joshua'
    [ivy:resolve] 		filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='commons-logging found='joshua'
    [ivy:resolve] 		filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='jaxen' found='joshua'
    [ivy:resolve] 		filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='jaxen found='joshua'
    [ivy:resolve] 		filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
    [ivy:resolve] 		filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='jung-algorithms found='joshua'
    [ivy:resolve] 		filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
    [ivy:resolve] 		filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='jung-api found='joshua'
    [ivy:resolve] 		filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
    [ivy:resolve] 		filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='jung-graph-impl found='joshua'
    [ivy:resolve] 		filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
    [ivy:resolve] 		filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='jung-visualization found='joshua'
    [ivy:resolve] 		filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='org.apache.hadoop' found='joshua'
    [ivy:resolve] 		filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='hadoop-core found='joshua'
    [ivy:resolve] 		filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='org.testng' found='joshua'
    [ivy:resolve] 		filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='testng found='joshua'
    [ivy:resolve] 		filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='org.mockito' found='joshua'
    [ivy:resolve] 		filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='mockito-all found='joshua'
    [ivy:resolve] 		filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='pmd' found='joshua'
    [ivy:resolve] 		filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='pmd found='joshua'
    [ivy:resolve] 		filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='junit' found='joshua'
    [ivy:resolve] 		filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='junit found='joshua'
    [ivy:resolve] 
    [ivy:resolve] :: USE VERBOSE OR DEBUG MESSAGE LEVEL FOR MORE DETAILS
    
    BUILD FAILED
    /Users/post/code/joshua2/build.xml:168: The following error occurred while executing this line:
    /Users/post/code/joshua2/build.xml:408: impossible to resolve dependencies:
    	resolve failed - see output for details
    
    Total time: 2 seconds

CC: @lukeorland

Yes, I'll fix that tonight.
On Wed, May 27, 2015 at 10:01 AM Matt Post <notifications@github.com> wrote:

> When I clone a fresh copy of Joshua and type "ant", I get the following
> ivy-related errors. @lukeorland <https://github.com/lukeorland>, can you
> look into this ASAP?
>
> $ git clone git@github.com:joshua-decoder/joshua joshua2
> $ cd joshua2
> $ export JOSHUA=$(pwd)
> $ ant
> Buildfile: /Users/post/code/joshua2/build.xml
>   [taskdef] Could not load definitions from resource org/doxygen/tools/antlib.xml. It could not be found.
>   [taskdef] Could not load definitions from resource testngtasks. It could not be found.
>
> setup:
>   [taskdef] Could not load definitions from resource org/doxygen/tools/antlib.xml. It could not be found.
>   [taskdef] Could not load definitions from resource testngtasks. It could not be found.
>
> check-joshua-home:
>      [echo] JOSHUA = /Users/post/code/joshua2 basedir = /Users/post/code/joshua2
>
> download-ivy:
>       [get] Destination already exists (skipping): /Users/post/code/joshua2/lib/ivy.jar
>
> init-ivy:
>
> resolve:
> [ivy:configure] :: Ivy 2.2.0 - 20100923230623 :: http://ant.apache.org/ivy/ ::
> [ivy:configure] :: loading settings :: file = /Users/post/code/joshua2/lib/ivysettings.xml
> [ivy:resolve] :: resolving dependencies :: joshua#joshua;working@fragment.local
> [ivy:resolve]   confs: [default]
> [ivy:resolve] :: resolution report :: resolve 982ms :: artifacts dl 1ms
>     ---------------------------------------------------------------------
>     |                  |            modules            ||   artifacts   |
>     |       conf       | number| search|dwnlded|evicted|| number|dwnlded|
>     ---------------------------------------------------------------------
>     |      default     |   15  |   0   |   0   |   0   ||   0   |   0   |
>     ---------------------------------------------------------------------
> [ivy:resolve]
> [ivy:resolve] :: problems summary ::
> [ivy:resolve] :::: WARNINGS
> [ivy:resolve]       ::::::::::::::::::::::::::::::::::::::::::::::
> [ivy:resolve]       ::          UNRESOLVED DEPENDENCIES         ::
> [ivy:resolve]       ::::::::::::::::::::::::::::::::::::::::::::::
> [ivy:resolve]       :: net.sourceforge.ant-doxygen#ant-doxygen;1.6.1: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='net.sourceforge.ant-doxygen' found='joshua'; bad module name: expected='ant-doxygen' found='joshua';
> [ivy:resolve]       :: asm#asm;3.1: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='asm' found='joshua'; bad module name: expected='asm' found='joshua';
> [ivy:resolve]       :: com.amazonaws#aws-java-sdk;1.1.3: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='com.amazonaws' found='joshua'; bad module name: expected='aws-java-sdk' found='joshua';
> [ivy:resolve]       :: org.apache.commons#commons-cli;1.2: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='org.apache.commons' found='joshua'; bad module name: expected='commons-cli' found='joshua';
> [ivy:resolve]       :: commons-logging#commons-logging;1.1.1: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='commons-logging' found='joshua'; bad module name: expected='commons-logging' found='joshua';
> [ivy:resolve]       :: jaxen#jaxen;1.1.1: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='jaxen' found='joshua'; bad module name: expected='jaxen' found='joshua';
> [ivy:resolve]       :: net.sf.jung#jung-algorithms;2.0: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='net.sf.jung' found='joshua'; bad module name: expected='jung-algorithms' found='joshua';
> [ivy:resolve]       :: net.sf.jung#jung-api;2.0: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='net.sf.jung' found='joshua'; bad module name: expected='jung-api' found='joshua';
> [ivy:resolve]       :: net.sf.jung#jung-graph-impl;2.0: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='net.sf.jung' found='joshua'; bad module name: expected='jung-graph-impl' found='joshua';
> [ivy:resolve]       :: net.sf.jung#jung-visualization;2.0: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='net.sf.jung' found='joshua'; bad module name: expected='jung-visualization' found='joshua';
> [ivy:resolve]       :: org.apache.hadoop#hadoop-core;0.20.203.0: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='org.apache.hadoop' found='joshua'; bad module name: expected='hadoop-core' found='joshua';
> [ivy:resolve]       :: org.testng#testng;6.7: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='org.testng' found='joshua'; bad module name: expected='testng' found='joshua';
> [ivy:resolve]       :: org.mockito#mockito-all;1.9.5: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='org.mockito' found='joshua'; bad module name: expected='mockito-all' found='joshua';
> [ivy:resolve]       :: pmd#pmd;4.2.5: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='pmd' found='joshua'; bad module name: expected='pmd' found='joshua';
> [ivy:resolve]       :: junit#junit;4.10: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='junit' found='joshua'; bad module name: expected='junit' found='joshua';
> [ivy:resolve]       ::::::::::::::::::::::::::::::::::::::::::::::
> [ivy:resolve] :::: ERRORS
> [ivy:resolve]       filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='net.sourceforge.ant-doxygen' found='joshua'
> [ivy:resolve]       filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='ant-doxygen found='joshua'
> [ivy:resolve]       filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='asm' found='joshua'
> [ivy:resolve]       filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='asm found='joshua'
> [ivy:resolve]       filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='com.amazonaws' found='joshua'
> [ivy:resolve]       filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='aws-java-sdk found='joshua'
> [ivy:resolve]       filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='org.apache.commons' found='joshua'
> [ivy:resolve]       filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='commons-cli found='joshua'
> [ivy:resolve]       filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='commons-logging' found='joshua'
> [ivy:resolve]       filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='commons-logging found='joshua'
> [ivy:resolve]       filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='jaxen' found='joshua'
> [ivy:resolve]       filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='jaxen found='joshua'
> [ivy:resolve]       filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
> [ivy:resolve]       filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='jung-algorithms found='joshua'
> [ivy:resolve]       filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
> [ivy:resolve]       filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='jung-api found='joshua'
> [ivy:resolve]       filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
> [ivy:resolve]       filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='jung-graph-impl found='joshua'
> [ivy:resolve]       filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
> [ivy:resolve]       filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='jung-visualization found='joshua'
> [ivy:resolve]       filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='org.apache.hadoop' found='joshua'
> [ivy:resolve]       filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='hadoop-core found='joshua'
> [ivy:resolve]       filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='org.testng' found='joshua'
> [ivy:resolve]       filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='testng found='joshua'
> [ivy:resolve]       filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='org.mockito' found='joshua'
> [ivy:resolve]       filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='mockito-all found='joshua'
> [ivy:resolve]       filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='pmd' found='joshua'
> [ivy:resolve]       filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='pmd found='joshua'
> [ivy:resolve]       filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='junit' found='joshua'
> [ivy:resolve]       filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='junit found='joshua'
> [ivy:resolve]
> [ivy:resolve] :: USE VERBOSE OR DEBUG MESSAGE LEVEL FOR MORE DETAILS
>
> BUILD FAILED
> /Users/post/code/joshua2/build.xml:168: The following error occurred while executing this line:
> /Users/post/code/joshua2/build.xml:408: impossible to resolve dependencies:
>     resolve failed - see output for details
>
> Total time: 2 seconds
>
> CC: @lukeorland <https://github.com/lukeorland>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joshua-decoder/joshua/issues/200>.
>

Is there a quick-fix or workaround I can use in the meantime, or will this require you to dig into it? (Trying to run a tutorial)


> On May 27, 2015, at 10:07 AM, Luke Orland <notifications@github.com> wrote:
> 
> Yes, I'll fix that tonight.
> On Wed, May 27, 2015 at 10:01 AM Matt Post <notifications@github.com> wrote:
> 
> > When I clone a fresh copy of Joshua and type "ant", I get the following
> > ivy-related errors. @lukeorland <https://github.com/lukeorland>, can you
> > look into this ASAP?
> >
> > $ git clone git@github.com:joshua-decoder/joshua joshua2
> > $ cd joshua2
> > $ export JOSHUA=$(pwd)
> > $ ant
> > Buildfile: /Users/post/code/joshua2/build.xml
> > [taskdef] Could not load definitions from resource org/doxygen/tools/antlib.xml. It could not be found.
> > [taskdef] Could not load definitions from resource testngtasks. It could not be found.
> >
> > setup:
> > [taskdef] Could not load definitions from resource org/doxygen/tools/antlib.xml. It could not be found.
> > [taskdef] Could not load definitions from resource testngtasks. It could not be found.
> >
> > check-joshua-home:
> > [echo] JOSHUA = /Users/post/code/joshua2 basedir = /Users/post/code/joshua2
> >
> > download-ivy:
> > [get] Destination already exists (skipping): /Users/post/code/joshua2/lib/ivy.jar
> >
> > init-ivy:
> >
> > resolve:
> > [ivy:configure] :: Ivy 2.2.0 - 20100923230623 :: http://ant.apache.org/ivy/ ::
> > [ivy:configure] :: loading settings :: file = /Users/post/code/joshua2/lib/ivysettings.xml
> > [ivy:resolve] :: resolving dependencies :: joshua#joshua;working@fragment.local
> > [ivy:resolve] confs: [default]
> > [ivy:resolve] :: resolution report :: resolve 982ms :: artifacts dl 1ms
> > ---------------------------------------------------------------------
> > | | modules || artifacts |
> > | conf | number| search|dwnlded|evicted|| number|dwnlded|
> > ---------------------------------------------------------------------
> > | default | 15 | 0 | 0 | 0 || 0 | 0 |
> > ---------------------------------------------------------------------
> > [ivy:resolve]
> > [ivy:resolve] :: problems summary ::
> > [ivy:resolve] :::: WARNINGS
> > [ivy:resolve] ::::::::::::::::::::::::::::::::::::::::::::::
> > [ivy:resolve] :: UNRESOLVED DEPENDENCIES ::
> > [ivy:resolve] ::::::::::::::::::::::::::::::::::::::::::::::
> > [ivy:resolve] :: net.sourceforge.ant-doxygen#ant-doxygen;1.6.1: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='net.sourceforge.ant-doxygen' found='joshua'; bad module name: expected='ant-doxygen' found='joshua';
> > [ivy:resolve] :: asm#asm;3.1: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='asm' found='joshua'; bad module name: expected='asm' found='joshua';
> > [ivy:resolve] :: com.amazonaws#aws-java-sdk;1.1.3: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='com.amazonaws' found='joshua'; bad module name: expected='aws-java-sdk' found='joshua';
> > [ivy:resolve] :: org.apache.commons#commons-cli;1.2: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='org.apache.commons' found='joshua'; bad module name: expected='commons-cli' found='joshua';
> > [ivy:resolve] :: commons-logging#commons-logging;1.1.1: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='commons-logging' found='joshua'; bad module name: expected='commons-logging' found='joshua';
> > [ivy:resolve] :: jaxen#jaxen;1.1.1: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='jaxen' found='joshua'; bad module name: expected='jaxen' found='joshua';
> > [ivy:resolve] :: net.sf.jung#jung-algorithms;2.0: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='net.sf.jung' found='joshua'; bad module name: expected='jung-algorithms' found='joshua';
> > [ivy:resolve] :: net.sf.jung#jung-api;2.0: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='net.sf.jung' found='joshua'; bad module name: expected='jung-api' found='joshua';
> > [ivy:resolve] :: net.sf.jung#jung-graph-impl;2.0: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='net.sf.jung' found='joshua'; bad module name: expected='jung-graph-impl' found='joshua';
> > [ivy:resolve] :: net.sf.jung#jung-visualization;2.0: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='net.sf.jung' found='joshua'; bad module name: expected='jung-visualization' found='joshua';
> > [ivy:resolve] :: org.apache.hadoop#hadoop-core;0.20.203.0: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='org.apache.hadoop' found='joshua'; bad module name: expected='hadoop-core' found='joshua';
> > [ivy:resolve] :: org.testng#testng;6.7: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='org.testng' found='joshua'; bad module name: expected='testng' found='joshua';
> > [ivy:resolve] :: org.mockito#mockito-all;1.9.5: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='org.mockito' found='joshua'; bad module name: expected='mockito-all' found='joshua';
> > [ivy:resolve] :: pmd#pmd;4.2.5: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='pmd' found='joshua'; bad module name: expected='pmd' found='joshua';
> > [ivy:resolve] :: junit#junit;4.10: java.text.ParseException: inconsistent module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='junit' found='joshua'; bad module name: expected='junit' found='joshua';
> > [ivy:resolve] ::::::::::::::::::::::::::::::::::::::::::::::
> > [ivy:resolve] :::: ERRORS
> > [ivy:resolve] filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='net.sourceforge.ant-doxygen' found='joshua'
> > [ivy:resolve] filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='ant-doxygen found='joshua'
> > [ivy:resolve] filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='asm' found='joshua'
> > [ivy:resolve] filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='asm found='joshua'
> > [ivy:resolve] filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='com.amazonaws' found='joshua'
> > [ivy:resolve] filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='aws-java-sdk found='joshua'
> > [ivy:resolve] filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='org.apache.commons' found='joshua'
> > [ivy:resolve] filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='commons-cli found='joshua'
> > [ivy:resolve] filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='commons-logging' found='joshua'
> > [ivy:resolve] filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='commons-logging found='joshua'
> > [ivy:resolve] filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='jaxen' found='joshua'
> > [ivy:resolve] filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='jaxen found='joshua'
> > [ivy:resolve] filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
> > [ivy:resolve] filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='jung-algorithms found='joshua'
> > [ivy:resolve] filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
> > [ivy:resolve] filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='jung-api found='joshua'
> > [ivy:resolve] filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
> > [ivy:resolve] filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='jung-graph-impl found='joshua'
> > [ivy:resolve] filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
> > [ivy:resolve] filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='jung-visualization found='joshua'
> > [ivy:resolve] filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='org.apache.hadoop' found='joshua'
> > [ivy:resolve] filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='hadoop-core found='joshua'
> > [ivy:resolve] filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='org.testng' found='joshua'
> > [ivy:resolve] filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='testng found='joshua'
> > [ivy:resolve] filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='org.mockito' found='joshua'
> > [ivy:resolve] filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='mockito-all found='joshua'
> > [ivy:resolve] filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='pmd' found='joshua'
> > [ivy:resolve] filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='pmd found='joshua'
> > [ivy:resolve] filesystem: bad organisation found in /Users/post/code/joshua2/lib/ivy.xml: expected='junit' found='joshua'
> > [ivy:resolve] filesystem: bad module name found in /Users/post/code/joshua2/lib/ivy.xml: expected='junit found='joshua'
> > [ivy:resolve]
> > [ivy:resolve] :: USE VERBOSE OR DEBUG MESSAGE LEVEL FOR MORE DETAILS
> >
> > BUILD FAILED
> > /Users/post/code/joshua2/build.xml:168: The following error occurred while executing this line:
> > /Users/post/code/joshua2/build.xml:408: impossible to resolve dependencies:
> > resolve failed - see output for details
> >
> > Total time: 2 seconds
> >
> > CC: @lukeorland <https://github.com/lukeorland>
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/joshua-decoder/joshua/issues/200>.
> >
> —
> Reply to this email directly or view it on GitHub <https://github.com/joshua-decoder/joshua/issues/200#issuecomment-105924405>.
>

Maybe copy the lib dir from another pre-existing clone of Joshua
On Wed, May 27, 2015 at 10:08 AM Matt Post <notifications@github.com> wrote:

> Is there a quick-fix or workaround I can use in the meantime, or will this
> require you to dig into it? (Trying to run a tutorial)
>
>
> > On May 27, 2015, at 10:07 AM, Luke Orland <notifications@github.com>
> wrote:
> >
> > Yes, I'll fix that tonight.
> > On Wed, May 27, 2015 at 10:01 AM Matt Post <notifications@github.com>
> wrote:
> >
> > > When I clone a fresh copy of Joshua and type "ant", I get the following
> > > ivy-related errors. @lukeorland <https://github.com/lukeorland>, can
> you
> > > look into this ASAP?
> > >
> > > $ git clone git@github.com:joshua-decoder/joshua joshua2
> > > $ cd joshua2
> > > $ export JOSHUA=$(pwd)
> > > $ ant
> > > Buildfile: /Users/post/code/joshua2/build.xml
> > > [taskdef] Could not load definitions from resource
> org/doxygen/tools/antlib.xml. It could not be found.
> > > [taskdef] Could not load definitions from resource testngtasks. It
> could not be found.
> > >
> > > setup:
> > > [taskdef] Could not load definitions from resource
> org/doxygen/tools/antlib.xml. It could not be found.
> > > [taskdef] Could not load definitions from resource testngtasks. It
> could not be found.
> > >
> > > check-joshua-home:
> > > [echo] JOSHUA = /Users/post/code/joshua2 basedir =
> /Users/post/code/joshua2
> > >
> > > download-ivy:
> > > [get] Destination already exists (skipping):
> /Users/post/code/joshua2/lib/ivy.jar
> > >
> > > init-ivy:
> > >
> > > resolve:
> > > [ivy:configure] :: Ivy 2.2.0 - 20100923230623 ::
> http://ant.apache.org/ivy/ ::
> > > [ivy:configure] :: loading settings :: file =
> /Users/post/code/joshua2/lib/ivysettings.xml
> > > [ivy:resolve] :: resolving dependencies ::
> joshua#joshua;working@fragment.local
> > > [ivy:resolve] confs: [default]
> > > [ivy:resolve] :: resolution report :: resolve 982ms :: artifacts dl 1ms
> > > ---------------------------------------------------------------------
> > > | | modules || artifacts |
> > > | conf | number| search|dwnlded|evicted|| number|dwnlded|
> > > ---------------------------------------------------------------------
> > > | default | 15 | 0 | 0 | 0 || 0 | 0 |
> > > ---------------------------------------------------------------------
> > > [ivy:resolve]
> > > [ivy:resolve] :: problems summary ::
> > > [ivy:resolve] :::: WARNINGS
> > > [ivy:resolve] ::::::::::::::::::::::::::::::::::::::::::::::
> > > [ivy:resolve] :: UNRESOLVED DEPENDENCIES ::
> > > [ivy:resolve] ::::::::::::::::::::::::::::::::::::::::::::::
> > > [ivy:resolve] :: net.sourceforge.ant-doxygen#ant-doxygen;1.6.1:
> java.text.ParseException: inconsistent module descriptor file found in
> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
> expected='net.sourceforge.ant-doxygen' found='joshua'; bad module name:
> expected='ant-doxygen' found='joshua';
> > > [ivy:resolve] :: asm#asm;3.1: java.text.ParseException: inconsistent
> module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad
> organisation: expected='asm' found='joshua'; bad module name:
> expected='asm' found='joshua';
> > > [ivy:resolve] :: com.amazonaws#aws-java-sdk;1.1.3:
> java.text.ParseException: inconsistent module descriptor file found in
> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
> expected='com.amazonaws' found='joshua'; bad module name:
> expected='aws-java-sdk' found='joshua';
> > > [ivy:resolve] :: org.apache.commons#commons-cli;1.2:
> java.text.ParseException: inconsistent module descriptor file found in
> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
> expected='org.apache.commons' found='joshua'; bad module name:
> expected='commons-cli' found='joshua';
> > > [ivy:resolve] :: commons-logging#commons-logging;1.1.1:
> java.text.ParseException: inconsistent module descriptor file found in
> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
> expected='commons-logging' found='joshua'; bad module name:
> expected='commons-logging' found='joshua';
> > > [ivy:resolve] :: jaxen#jaxen;1.1.1: java.text.ParseException:
> inconsistent module descriptor file found in
> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='jaxen'
> found='joshua'; bad module name: expected='jaxen' found='joshua';
> > > [ivy:resolve] :: net.sf.jung#jung-algorithms;2.0:
> java.text.ParseException: inconsistent module descriptor file found in
> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
> expected='net.sf.jung' found='joshua'; bad module name:
> expected='jung-algorithms' found='joshua';
> > > [ivy:resolve] :: net.sf.jung#jung-api;2.0: java.text.ParseException:
> inconsistent module descriptor file found in
> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
> expected='net.sf.jung' found='joshua'; bad module name: expected='jung-api'
> found='joshua';
> > > [ivy:resolve] :: net.sf.jung#jung-graph-impl;2.0:
> java.text.ParseException: inconsistent module descriptor file found in
> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
> expected='net.sf.jung' found='joshua'; bad module name:
> expected='jung-graph-impl' found='joshua';
> > > [ivy:resolve] :: net.sf.jung#jung-visualization;2.0:
> java.text.ParseException: inconsistent module descriptor file found in
> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
> expected='net.sf.jung' found='joshua'; bad module name:
> expected='jung-visualization' found='joshua';
> > > [ivy:resolve] :: org.apache.hadoop#hadoop-core;0.20.203.0:
> java.text.ParseException: inconsistent module descriptor file found in
> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
> expected='org.apache.hadoop' found='joshua'; bad module name:
> expected='hadoop-core' found='joshua';
> > > [ivy:resolve] :: org.testng#testng;6.7: java.text.ParseException:
> inconsistent module descriptor file found in
> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
> expected='org.testng' found='joshua'; bad module name: expected='testng'
> found='joshua';
> > > [ivy:resolve] :: org.mockito#mockito-all;1.9.5:
> java.text.ParseException: inconsistent module descriptor file found in
> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
> expected='org.mockito' found='joshua'; bad module name:
> expected='mockito-all' found='joshua';
> > > [ivy:resolve] :: pmd#pmd;4.2.5: java.text.ParseException: inconsistent
> module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad
> organisation: expected='pmd' found='joshua'; bad module name:
> expected='pmd' found='joshua';
> > > [ivy:resolve] :: junit#junit;4.10: java.text.ParseException:
> inconsistent module descriptor file found in
> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='junit'
> found='joshua'; bad module name: expected='junit' found='joshua';
> > > [ivy:resolve] ::::::::::::::::::::::::::::::::::::::::::::::
> > > [ivy:resolve] :::: ERRORS
> > > [ivy:resolve] filesystem: bad organisation found in
> /Users/post/code/joshua2/lib/ivy.xml:
> expected='net.sourceforge.ant-doxygen' found='joshua'
> > > [ivy:resolve] filesystem: bad module name found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='ant-doxygen found='joshua'
> > > [ivy:resolve] filesystem: bad organisation found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='asm' found='joshua'
> > > [ivy:resolve] filesystem: bad module name found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='asm found='joshua'
> > > [ivy:resolve] filesystem: bad organisation found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='com.amazonaws'
> found='joshua'
> > > [ivy:resolve] filesystem: bad module name found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='aws-java-sdk found='joshua'
> > > [ivy:resolve] filesystem: bad organisation found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='org.apache.commons'
> found='joshua'
> > > [ivy:resolve] filesystem: bad module name found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='commons-cli found='joshua'
> > > [ivy:resolve] filesystem: bad organisation found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='commons-logging'
> found='joshua'
> > > [ivy:resolve] filesystem: bad module name found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='commons-logging
> found='joshua'
> > > [ivy:resolve] filesystem: bad organisation found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='jaxen' found='joshua'
> > > [ivy:resolve] filesystem: bad module name found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='jaxen found='joshua'
> > > [ivy:resolve] filesystem: bad organisation found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
> > > [ivy:resolve] filesystem: bad module name found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='jung-algorithms
> found='joshua'
> > > [ivy:resolve] filesystem: bad organisation found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
> > > [ivy:resolve] filesystem: bad module name found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='jung-api found='joshua'
> > > [ivy:resolve] filesystem: bad organisation found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
> > > [ivy:resolve] filesystem: bad module name found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='jung-graph-impl
> found='joshua'
> > > [ivy:resolve] filesystem: bad organisation found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
> > > [ivy:resolve] filesystem: bad module name found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='jung-visualization
> found='joshua'
> > > [ivy:resolve] filesystem: bad organisation found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='org.apache.hadoop'
> found='joshua'
> > > [ivy:resolve] filesystem: bad module name found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='hadoop-core found='joshua'
> > > [ivy:resolve] filesystem: bad organisation found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='org.testng' found='joshua'
> > > [ivy:resolve] filesystem: bad module name found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='testng found='joshua'
> > > [ivy:resolve] filesystem: bad organisation found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='org.mockito' found='joshua'
> > > [ivy:resolve] filesystem: bad module name found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='mockito-all found='joshua'
> > > [ivy:resolve] filesystem: bad organisation found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='pmd' found='joshua'
> > > [ivy:resolve] filesystem: bad module name found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='pmd found='joshua'
> > > [ivy:resolve] filesystem: bad organisation found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='junit' found='joshua'
> > > [ivy:resolve] filesystem: bad module name found in
> /Users/post/code/joshua2/lib/ivy.xml: expected='junit found='joshua'
> > > [ivy:resolve]
> > > [ivy:resolve] :: USE VERBOSE OR DEBUG MESSAGE LEVEL FOR MORE DETAILS
> > >
> > > BUILD FAILED
> > > /Users/post/code/joshua2/build.xml:168: The following error occurred
> while executing this line:
> > > /Users/post/code/joshua2/build.xml:408: impossible to resolve
> dependencies:
> > > resolve failed - see output for details
> > >
> > > Total time: 2 seconds
> > >
> > > CC: @lukeorland <https://github.com/lukeorland>
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <https://github.com/joshua-decoder/joshua/issues/200>.
> > >
> > —
> > Reply to this email directly or view it on GitHub <
> https://github.com/joshua-decoder/joshua/issues/200#issuecomment-105924405
> >.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joshua-decoder/joshua/issues/200#issuecomment-105925015>
> .
>

Or temporarily revert the merge commit that changed ivy settings.xml
On Wed, May 27, 2015 at 10:09 AM Luke Orland <orluke@gmail.com> wrote:

> Maybe copy the lib dir from another pre-existing clone of Joshua
> On Wed, May 27, 2015 at 10:08 AM Matt Post <notifications@github.com>
> wrote:
>
>> Is there a quick-fix or workaround I can use in the meantime, or will
>> this require you to dig into it? (Trying to run a tutorial)
>>
>>
>> > On May 27, 2015, at 10:07 AM, Luke Orland <notifications@github.com>
>> wrote:
>> >
>> > Yes, I'll fix that tonight.
>> > On Wed, May 27, 2015 at 10:01 AM Matt Post <notifications@github.com>
>> wrote:
>> >
>> > > When I clone a fresh copy of Joshua and type "ant", I get the
>> following
>> > > ivy-related errors. @lukeorland <https://github.com/lukeorland>, can
>> you
>> > > look into this ASAP?
>> > >
>> > > $ git clone git@github.com:joshua-decoder/joshua joshua2
>> > > $ cd joshua2
>> > > $ export JOSHUA=$(pwd)
>> > > $ ant
>> > > Buildfile: /Users/post/code/joshua2/build.xml
>> > > [taskdef] Could not load definitions from resource
>> org/doxygen/tools/antlib.xml. It could not be found.
>> > > [taskdef] Could not load definitions from resource testngtasks. It
>> could not be found.
>> > >
>> > > setup:
>> > > [taskdef] Could not load definitions from resource
>> org/doxygen/tools/antlib.xml. It could not be found.
>> > > [taskdef] Could not load definitions from resource testngtasks. It
>> could not be found.
>> > >
>> > > check-joshua-home:
>> > > [echo] JOSHUA = /Users/post/code/joshua2 basedir =
>> /Users/post/code/joshua2
>> > >
>> > > download-ivy:
>> > > [get] Destination already exists (skipping):
>> /Users/post/code/joshua2/lib/ivy.jar
>> > >
>> > > init-ivy:
>> > >
>> > > resolve:
>> > > [ivy:configure] :: Ivy 2.2.0 - 20100923230623 ::
>> http://ant.apache.org/ivy/ ::
>> > > [ivy:configure] :: loading settings :: file =
>> /Users/post/code/joshua2/lib/ivysettings.xml
>> > > [ivy:resolve] :: resolving dependencies ::
>> joshua#joshua;working@fragment.local
>> > > [ivy:resolve] confs: [default]
>> > > [ivy:resolve] :: resolution report :: resolve 982ms :: artifacts dl
>> 1ms
>> > > ---------------------------------------------------------------------
>> > > | | modules || artifacts |
>> > > | conf | number| search|dwnlded|evicted|| number|dwnlded|
>> > > ---------------------------------------------------------------------
>> > > | default | 15 | 0 | 0 | 0 || 0 | 0 |
>> > > ---------------------------------------------------------------------
>> > > [ivy:resolve]
>> > > [ivy:resolve] :: problems summary ::
>> > > [ivy:resolve] :::: WARNINGS
>> > > [ivy:resolve] ::::::::::::::::::::::::::::::::::::::::::::::
>> > > [ivy:resolve] :: UNRESOLVED DEPENDENCIES ::
>> > > [ivy:resolve] ::::::::::::::::::::::::::::::::::::::::::::::
>> > > [ivy:resolve] :: net.sourceforge.ant-doxygen#ant-doxygen;1.6.1:
>> java.text.ParseException: inconsistent module descriptor file found in
>> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
>> expected='net.sourceforge.ant-doxygen' found='joshua'; bad module name:
>> expected='ant-doxygen' found='joshua';
>> > > [ivy:resolve] :: asm#asm;3.1: java.text.ParseException: inconsistent
>> module descriptor file found in '/Users/post/code/joshua2/lib/ivy.xml': bad
>> organisation: expected='asm' found='joshua'; bad module name:
>> expected='asm' found='joshua';
>> > > [ivy:resolve] :: com.amazonaws#aws-java-sdk;1.1.3:
>> java.text.ParseException: inconsistent module descriptor file found in
>> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
>> expected='com.amazonaws' found='joshua'; bad module name:
>> expected='aws-java-sdk' found='joshua';
>> > > [ivy:resolve] :: org.apache.commons#commons-cli;1.2:
>> java.text.ParseException: inconsistent module descriptor file found in
>> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
>> expected='org.apache.commons' found='joshua'; bad module name:
>> expected='commons-cli' found='joshua';
>> > > [ivy:resolve] :: commons-logging#commons-logging;1.1.1:
>> java.text.ParseException: inconsistent module descriptor file found in
>> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
>> expected='commons-logging' found='joshua'; bad module name:
>> expected='commons-logging' found='joshua';
>> > > [ivy:resolve] :: jaxen#jaxen;1.1.1: java.text.ParseException:
>> inconsistent module descriptor file found in
>> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='jaxen'
>> found='joshua'; bad module name: expected='jaxen' found='joshua';
>> > > [ivy:resolve] :: net.sf.jung#jung-algorithms;2.0:
>> java.text.ParseException: inconsistent module descriptor file found in
>> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
>> expected='net.sf.jung' found='joshua'; bad module name:
>> expected='jung-algorithms' found='joshua';
>> > > [ivy:resolve] :: net.sf.jung#jung-api;2.0: java.text.ParseException:
>> inconsistent module descriptor file found in
>> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
>> expected='net.sf.jung' found='joshua'; bad module name: expected='jung-api'
>> found='joshua';
>> > > [ivy:resolve] :: net.sf.jung#jung-graph-impl;2.0:
>> java.text.ParseException: inconsistent module descriptor file found in
>> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
>> expected='net.sf.jung' found='joshua'; bad module name:
>> expected='jung-graph-impl' found='joshua';
>> > > [ivy:resolve] :: net.sf.jung#jung-visualization;2.0:
>> java.text.ParseException: inconsistent module descriptor file found in
>> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
>> expected='net.sf.jung' found='joshua'; bad module name:
>> expected='jung-visualization' found='joshua';
>> > > [ivy:resolve] :: org.apache.hadoop#hadoop-core;0.20.203.0:
>> java.text.ParseException: inconsistent module descriptor file found in
>> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
>> expected='org.apache.hadoop' found='joshua'; bad module name:
>> expected='hadoop-core' found='joshua';
>> > > [ivy:resolve] :: org.testng#testng;6.7: java.text.ParseException:
>> inconsistent module descriptor file found in
>> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
>> expected='org.testng' found='joshua'; bad module name: expected='testng'
>> found='joshua';
>> > > [ivy:resolve] :: org.mockito#mockito-all;1.9.5:
>> java.text.ParseException: inconsistent module descriptor file found in
>> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation:
>> expected='org.mockito' found='joshua'; bad module name:
>> expected='mockito-all' found='joshua';
>> > > [ivy:resolve] :: pmd#pmd;4.2.5: java.text.ParseException:
>> inconsistent module descriptor file found in
>> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='pmd'
>> found='joshua'; bad module name: expected='pmd' found='joshua';
>> > > [ivy:resolve] :: junit#junit;4.10: java.text.ParseException:
>> inconsistent module descriptor file found in
>> '/Users/post/code/joshua2/lib/ivy.xml': bad organisation: expected='junit'
>> found='joshua'; bad module name: expected='junit' found='joshua';
>> > > [ivy:resolve] ::::::::::::::::::::::::::::::::::::::::::::::
>> > > [ivy:resolve] :::: ERRORS
>> > > [ivy:resolve] filesystem: bad organisation found in
>> /Users/post/code/joshua2/lib/ivy.xml:
>> expected='net.sourceforge.ant-doxygen' found='joshua'
>> > > [ivy:resolve] filesystem: bad module name found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='ant-doxygen found='joshua'
>> > > [ivy:resolve] filesystem: bad organisation found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='asm' found='joshua'
>> > > [ivy:resolve] filesystem: bad module name found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='asm found='joshua'
>> > > [ivy:resolve] filesystem: bad organisation found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='com.amazonaws'
>> found='joshua'
>> > > [ivy:resolve] filesystem: bad module name found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='aws-java-sdk found='joshua'
>> > > [ivy:resolve] filesystem: bad organisation found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='org.apache.commons'
>> found='joshua'
>> > > [ivy:resolve] filesystem: bad module name found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='commons-cli found='joshua'
>> > > [ivy:resolve] filesystem: bad organisation found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='commons-logging'
>> found='joshua'
>> > > [ivy:resolve] filesystem: bad module name found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='commons-logging
>> found='joshua'
>> > > [ivy:resolve] filesystem: bad organisation found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='jaxen' found='joshua'
>> > > [ivy:resolve] filesystem: bad module name found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='jaxen found='joshua'
>> > > [ivy:resolve] filesystem: bad organisation found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
>> > > [ivy:resolve] filesystem: bad module name found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='jung-algorithms
>> found='joshua'
>> > > [ivy:resolve] filesystem: bad organisation found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
>> > > [ivy:resolve] filesystem: bad module name found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='jung-api found='joshua'
>> > > [ivy:resolve] filesystem: bad organisation found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
>> > > [ivy:resolve] filesystem: bad module name found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='jung-graph-impl
>> found='joshua'
>> > > [ivy:resolve] filesystem: bad organisation found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='net.sf.jung' found='joshua'
>> > > [ivy:resolve] filesystem: bad module name found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='jung-visualization
>> found='joshua'
>> > > [ivy:resolve] filesystem: bad organisation found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='org.apache.hadoop'
>> found='joshua'
>> > > [ivy:resolve] filesystem: bad module name found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='hadoop-core found='joshua'
>> > > [ivy:resolve] filesystem: bad organisation found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='org.testng' found='joshua'
>> > > [ivy:resolve] filesystem: bad module name found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='testng found='joshua'
>> > > [ivy:resolve] filesystem: bad organisation found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='org.mockito' found='joshua'
>> > > [ivy:resolve] filesystem: bad module name found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='mockito-all found='joshua'
>> > > [ivy:resolve] filesystem: bad organisation found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='pmd' found='joshua'
>> > > [ivy:resolve] filesystem: bad module name found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='pmd found='joshua'
>> > > [ivy:resolve] filesystem: bad organisation found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='junit' found='joshua'
>> > > [ivy:resolve] filesystem: bad module name found in
>> /Users/post/code/joshua2/lib/ivy.xml: expected='junit found='joshua'
>> > > [ivy:resolve]
>> > > [ivy:resolve] :: USE VERBOSE OR DEBUG MESSAGE LEVEL FOR MORE DETAILS
>> > >
>> > > BUILD FAILED
>> > > /Users/post/code/joshua2/build.xml:168: The following error occurred
>> while executing this line:
>> > > /Users/post/code/joshua2/build.xml:408: impossible to resolve
>> dependencies:
>> > > resolve failed - see output for details
>> > >
>> > > Total time: 2 seconds
>> > >
>> > > CC: @lukeorland <https://github.com/lukeorland>
>> > >
>> > > —
>> > > Reply to this email directly or view it on GitHub
>> > > <https://github.com/joshua-decoder/joshua/issues/200>.
>> > >
>> > —
>> > Reply to this email directly or view it on GitHub <
>> https://github.com/joshua-decoder/joshua/issues/200#issuecomment-105924405
>> >.
>> >
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/joshua-decoder/joshua/issues/200#issuecomment-105925015>
>> .
>>
>

--- a/lib/ivysettings.xml
+++ b/lib/ivysettings.xml
@@ -2,24 +2,17 @@
 <ivysettings >
   <settings defaultResolver="1" />
   <resolvers>
-    <chain name="1">
-
-      <chain name="1">
-        <filesystem name="filesystem">
-          <ivy pattern="${ivy.settings.dir}/ivy.xml"/>
-          <artifact pattern="${ivy.settings.dir}/[artifact]-[revision].[ext]"/>
-        </filesystem>
-      </chain>
-
-      <chain name="maven">
-        <ibiblio name="central" m2compatible="true"/>
-        <ibiblio name="tools.gbif.org" m2compatible="true" root="http://tools.gbif.org/maven/repository/" />
-        <packager name="roundup" buildRoot="${env.JOSHUA}/lib/packager/build" resourceCache="${env.JOSHUA}/lib/cache">
-          <ivy pattern="http://ivyroundup.googlecode.com/svn/trunk/repo/modules/[organisation]/[module]/[revision]/ivy.xml"/>
-          <artifact pattern="http://ivyroundup.googlecode.com/svn/trunk/repo/modules/[organisation]/[module]/[revision]/packager.xml"/>
-        </packager>
-      </chain>
-
+    <chain name="1" returnFirst="true" dual="true">
+      <filesystem name="filesystem">
+        <ivy pattern="${ivy.settings.dir}/cache/[organisation]/[module]/ivys/ivy-[revision].xml"/>
+        <artifact pattern="${ivy.settings.dir}/[artifact]-[revision].[ext]"/>
+      </filesystem>
+      <ibiblio name="central" m2compatible="true"/>
+      <ibiblio name="tools.gbif.org" m2compatible="true" root="http://tools.gbif.org/maven/repository/" />
+      <packager name="roundup" buildRoot="${env.JOSHUA}/lib/packager/build" resourceCache="${env.JOSHUA}/lib/cache">
+        <ivy pattern="http://ivyroundup.googlecode.com/svn/trunk/repo/modules/[organisation]/[module]/[revision]/ivy.xml"/>
+        <artifact pattern="http://ivyroundup.googlecode.com/svn/trunk/repo/modules/[organisation]/[module]/[revision]/packager.xml"/>
+      </packager>
     </chain>
   </resolvers>
 </ivysettings>

